tests/templates/kuttl/resources/10-assert.yaml.j2

```diff
@@ -11,6 +11,9 @@ metadata:
     app.kubernetes.io/managed-by: spark.stackable.tech_sparkapplication
 spec:
   containers:
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+    - name: vector
+{% endif %}
```

Comment:
You have changed to order of containers, I don't think this will work
This will however only be tested when the env `VECTOR_AGGREGATOR` is set, which happens on jenkins

Reply:
corrected